src/main/java/com/cosc436002fitzarr/brm/models/buildingriskassessment/BuildingRiskAssessment.java

```diff
@@ -1,107 +1,37 @@
 package com.cosc436002fitzarr.brm.models.buildingriskassessment;
 
-import com.cosc436002fitzarr.brm.enums.Status;
-import com.cosc436002fitzarr.brm.models.Entity;
-import com.cosc436002fitzarr.brm.models.EntityTrail;
-import com.cosc436002fitzarr.brm.models.riskassessment.Hazard;
-import com.cosc436002fitzarr.brm.models.riskassessment.Screener;
 import org.springframework.data.mongodb.core.mapping.Document;
-import java.time.LocalDateTime;
 import java.util.List;
 import java.util.Objects;
 
 @Document(collection = "buildingriskassessment")
-public class BuildingRiskAssessment implements Entity {
-    private String id;
-    private LocalDateTime createdAt;
-    private LocalDateTime updatedAt;
-    private List<EntityTrail> entityTrail;
-    private String publisherId;
-    private Long workOrder;
+public class BuildingRiskAssessment {
```

Comment:
I think we might need to change this model name to `BuildingRiskAssessments` since there can be multiple risk assessments at a building.